### PR TITLE
add support to double click to equip spirit books

### PR DESCRIPTION
**Pull request by matiascalegaris**

### CODE REVIEW

File: CODIGO/frmMain.frm [hunk 1]
```diff
@@ -3732,7 +3733,7 @@ Private Sub picInv_MouseMove(Button As Integer, Shift As Integer, x As Single, y
         Select Case ObjData(Inventario.ObjIndex(Slot)).ObjType
 
             Case eObjType.otWeapon
-                ObjLbl = Inventario.ItemName(Slot) & " (" & Inventario.Amount(Slot) & ")" & vbCrLf & "Daño: " & ObjData(Inventario.ObjIndex(Slot)).MinHit & "/" & ObjData(Inventario.ObjIndex(Slot)).MaxHit
+                ObjLbl = Inventario.ItemName(Slot) & " (" & Inventario.Amount(Slot) & ")" & vbCrLf & "Daño: " & ObjData(Inventario.OBJIndex(Slot)).MinHit & "/" & ObjData(Inventario.OBJIndex(Slot)).MaxHit
```
Comment (morgolock) [inline CODIGO/frmMain.frm:3736]: Can you please move this out to a new module `language_utils` or something like that?

We must include the `english` and `spanish` strings.


Lets avoid using string literals among the code from now on

**Review by morgolock — CHANGES_REQUESTED** [review]

File: CODIGO/frmMain.frm (hunk 1 above)
Comment (matiascalegaris) [reply CODIGO/frmMain.frm:3736]: I can but thats not really part of this pr, we should do it as dedicate effort for that

Comment (morgolock) [reply CODIGO/frmMain.frm:3736]: Ok, agreed out of the scope of this patch.

Lets rework that later.

**Review by morgolock — APPROVED** [review]
TODO: remove string literals to a localization module or file

**Pull request merged**